CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ this project adheres to
   multiple removals.
 - Updated review-database to 0.37.0, which fixes event filtering by multiple IP
   addresses for `ExternalDdos`, `MultiHostPortScan`, and `RdpBruteForce`.
+- Renamed `host_names` field to `hostnames` in Filter GraphQL schema to align
+  with database naming convention.
```

Comment:
Could you specify which GraphQL APIs or types are affected by this change, preferably by quoting them directly?

Including relevant details in the changelog would also help inform **users** about what they should be aware of from an **API perspective**.

Reply:
The rationale behind this is that we need to clearly inform users of what has changed from their perspective.

Reply:
@sehkone Thank you for the feedback. I've updated the changelog to specify the affected GraphQL types and APIs:

- `FilterInput`, `EventListFilterInput`, and `Filter` types
- GraphQL APIs: `insertFilter`, `replaceFilter`, `filter`, and `filterList`